tcks/microprofile-context-propagation/pom.xml

```diff
@@ -29,10 +29,6 @@
                     <dependenciesToScan>
                         <dependency>org.eclipse.microprofile.context-propagation:microprofile-context-propagation-tck</dependency>
                     </dependenciesToScan>
-                    <!-- To be removed once #3878 is implemented-->
-                    <excludes>
-                        <exclude>**/JTACDITest.java</exclude>
```

Comment:
@kenfinnigan FYI, we are going to be one test closer to passing TCKs ;-)

Reply:
woohoo!